u2f-gae-demo/README.md

```diff
@@ -0,0 +1,15 @@
+# U2F Demo Deployment
+
+To deploy https://crxjs-dot-u2fdemo.appspot.com and configure the appspot
+server to the package Chrome app, you have to:
```

Comment:
"the package Chrome app": I'm not sure what you mean here. Just to keep the terminology straight, there are:
1. The cryptotoken component extension, i.e. the extension that's built into Chrome.
2. The U2F extension, which we use for development.
3. The gnubbyd packaged app, which is used internally by Google, and not relevant here.

Which do you mean?